src/MonsterImage.tsx

```diff
@@ -44,7 +44,7 @@ export function MonsterImage({
         alt=""
         className={classNames("db img-shadow h-auto", {
           "o-0": state === "errored",
-          "img-crisp": imageType === "sprite",
+          "img-crisp": true,
```

Comment:
Pokémon 1009 Walking Wake and Pokémon 1010 Iron Leaves are still using the "HD" images, and shouldn't have `img-crisp` or they will look funny